### DISCUSSION TITLE
Fatal Error C1083

### DISCUSSION THREAD

**Discussion by khi-ayo** [FunkinCrew/Funkin#1574] [Compiling help]
I got this error after doing the 'lime test windows -debug' command (the last step to setting up the game according to the guide.)
How do I fix this?
<img width="656" alt="fatal error" src="https://user-images.githubusercontent.com/85705792/121596884-e40a1000-c9f4-11eb-9ab7-eed89905512c.PNG">

**Comment by aflacc** [comment]
did you install discord-rpc

**Comment by khi-ayo** [comment]
yeah i did
